com.aobuchow.themes.spectrum/css/default_overrides.css

```diff
@@ -359,3 +359,27 @@ ProgressIndicator {
   background-color: '#com-aobuchow-themes-spectrum-BACKGROUND_COLOR';
   color: #eeeeee;
 }
+
+
+/* ############################## Issue 137  ############################## */
+
+TabbedPropertyTitle > CLabel {
+	color: white;
+}
+
+TabbedPropertyTitle {
+	swt-backgroundGradientStart-color:   #2f2f2f;
```

Comment:
@PyvesB I was having trouble setting these colors to color extension points (eg. `'#com-aobuchow-themes-spectrum-BASE_COLOR'`). Have you encountered a similar issue with your theme by any chance?

Reply:
https://github.com/PyvesB/eclipse-planet-themes/blob/cd9e3d404e217198a6ffad1f4ce9fdeda73de307/planet-themes-plugin/css/planets.css#L95

^ I've tended to only add properties when I've spotted an explicit use-case, so I'm assuming that works. The casing of the property name is different from yours though, not sure whether it matters.